package.json

```diff
@@ -0,0 +1,13 @@
+{
+	"name": "signal_generator",
+	"version": "0.0.1",
+
+	"author": "Alvaro Souza",
+	"description": "Signal generator and processor",
+
+	"dependencies": {
+		"hyro_api_channel": "*",
```

Comment:
Missing dependencies.
Usually all packages required in the CMakeList.txt need to be in the dependency list of package.json